packages/react-components/react-combobox/src/components/Combobox/Combobox.test.tsx

```diff
@@ -133,6 +133,25 @@ describe('Combobox', () => {
     expect(queryByRole('listbox')).toBeNull();
   });
 
+  it('closes the popup on blur/outside click', () => {
```

Comment:
```suggestion
  it('closes the popup on blur/outside click after clicking on the expand icon', () => {
```

is this the problem you found?

Reply:
yup!